hazelcast/src/test/java/com/hazelcast/cache/impl/InvalidationEventQueueTest.java

```diff
@@ -28,15 +30,15 @@
 
     @Test
     public void itemsShouldBeOfferedCorrectly() throws InterruptedException, ExecutionException, TimeoutException {
-        final InvalidationEventQueue queue = new InvalidationEventQueue();
+        final BatchInvalidator.InvalidationQueue queue = new BatchInvalidator.InvalidationQueue();
```

Comment:
Could you add an import for `BatchInvalidator.InvalidationQueue`, would make this test easier to read (and reduce the number of changes).